scripts/validate-generated.sh

```diff
@@ -20,7 +20,10 @@ cp -a "${DIR}/.." "${T}/"
 (cd "${T}/" && go generate ./...)
 
 
-GENERATED_FILES=("pkg/openshift/certgen/templates/bindata.go")
+GENERATED_FILES=(
```

Comment:
need update to scripts/validate-go.sh as well?

Reply:
updated